container/webui/Dockerfile

```
@@ -9,6 +9,9 @@ RUN zypper ar -p 95 -f http://download.opensuse.org/repositories/devel:openQA/op
     zypper --gpg-auto-import-keys ref && \
     zypper in -y ca-certificates-mozilla curl && \
     zypper in -y --force-resolution openQA-local-db apache2 hostname which w3m && \
+    # The openQA needs rubygem:sass to precompile the assets, otherways there are errors during the tests eventually 
+    # But there is nothing wrong with the spec file because it does not require sass as the assets are precompiled as cache.tar
```

```suggestion
    # openQA might need rubygem:sass to compile assets in this context due to
    # unknown reasons even though the package should use the precompiled
    # assets from cache.tar
```

I also suggest to combine all three `zypper in` calls. Doing all in one step can be a tiny bit faster than three steps of installing